app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -68,8 +69,11 @@ class UserInfoAPIController extends BaseController {
         val message = "Could not find Authorization in header"
         Logger.error(message)
         Unauthorized(message)
-      } { token ⇒
-        userInfoGen.seeded(token).fold(
+      }{ token ⇒
+        val headerToken = headers.get("token")
```

Comment:
The token you want to work with is `token` in the line above - there won't be a `"token"` value in the headers (at least in the backend code calling this endpoint doesn't currently put one there)